api/DOMException.json

```diff
@@ -212,7 +212,7 @@
               "version_added": "4"
             },
             "ie": {
-              "version_added": null
+              "version_added": "10"
```

Comment:
Is this (api.DOMException.name) based on mdn-bcd-collector, or manual testing? If it's the collector, can you confirm it's correct? It seems a bit late for this old API...

Reply:
This was from the collector, I just checked it in manual testing and confirmed that `name` is not present in the object's prototype.  It's possible that this suffers the same issue as described in https://github.com/foolip/mdn-bcd-collector/issues/380.